platform/run_image.go

```diff
@@ -100,3 +62,39 @@ func byRegistry(reg string, images []string, checkReadAccess CheckReadAccess, ke
 	}
 	return ""
 }
+
+// GetRunImageForExport TODO
```

Comment:
hahaha you already found a way to outsmart the linter 😆

Reply:
Oops! Updated in 60603a37